blockkit/elements.py

```diff
@@ -717,3 +719,27 @@ def __init__(
         indent: Optional[int] = None,
     ):
         super().__init__(elements=elements, style=style, indent=indent)
+
+class FileInput(FocusableElement):
+    type: str = "file_input"
+    filetypes: Optional[List[str]] = Field(None)
+    maxfiles: Optional[int] = Field(None)
```

Comment:
I would suggest using Field(..., gt=0, le=10) instead of defining a custom validator.